addons/godot-next/resources/behavior.gd

```diff
@@ -0,0 +1,67 @@
+# author: xdgamestudios
+# license: MIT
+# description: This is an abstract base class for all Behaviors. 
```

Comment:
Explain *what* a Behavior class is. Example:

    # This is an abstract base "Behavior" class for use in the "Behaviors" node class.
    # "Behaviors" manages "Behavior" resources and calls notification methods that the Behavior implements.
    # Supported notifications include...
    # - _ready() -> void
    # - _process(p_delta: float) -> void
    # ....
    # Individual Behavior resources can be toggled on and off using the `enabled` property.
    # `owner` is a reference to the Behaviors node.

Feel free to tweak as you like.